src/hooks/wallets/useOnboard.ts

```diff
@@ -197,12 +197,8 @@ export const useInitOnboard = () => {
       onboard.state.actions.setWalletModules(supportedWallets)
     }
 
-    enableWallets()
-  }, [chain, onboard])
-
-  // Connect to the last connected wallet
-  useEffect(() => {
-    if (onboard && onboard.state.get().wallets.length === 0) {
```

Comment:
We still need to check that there is no active wallet (`onboard.state.get().wallets.length === 0`) before connecting again otherwise we get this warning:

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/2e030dd3-3dbf-4879-bd7e-cf161e7be440)

Reply:
Restored it. 👍